### DISCUSSION TITLE
Debug mode

### DISCUSSION THREAD

**Discussion by alabotski** [d4rken-org/sdmaid-se#976] [Q&A] [answered]
**Describe the bug**
I disabled debug mode but debug files are created

**To Reproduce**
1. I enable Debug mode
2. Set such settings [Screenshot_20240128-111044](https://github.com/d4rken-org/sdmaid-se/assets/439337/d9e9bdb6-1c43-4087-a9a1-b1b2b59af5dd)
3. Disable debug mode
4. I keep seeing empty debug files [Screenshot_20240128-111342](https://github.com/d4rken-org/sdmaid-se/assets/439337/afbfea84-8255-41a2-932f-99ef1026aa48)

**Expected behavior**
I want the global disabling of debug mode setting to be in the background

**Smartphone (please complete the following information):**
 - Device: Pixel 7 Pro
 - OS: Android 14
 - Root status Kitsune
 - SD Maid 2/SE version 0.19

**Debug log**
If you can reproduce the bug, record the event in [a debug log](https://github.com/d4rken-org/sdmaid-se/wiki/Bugs#debug-log).

**Screenshots/Video**
If applicable, add screenshots or a video if it helps to explain your problem.

**Additional context**

**Comment by d4rken** [comment, marked as answer]
Please don't enable settings you don't understand.

You need to disable "Dry-Run mode"
